tasks/gulp/compile-assets.js

```diff
@@ -143,8 +143,8 @@ gulp.task('js:compile', () => {
   // for dist/ folder we only want compiled 'all.js' file
   let srcFiles = isDist ? configPaths.src + 'all.js' : configPaths.src + '**/*.js'
   return gulp.src([
-    '!' + configPaths.src + '**/*.test.js',
-    srcFiles
+    srcFiles,
+    '!' + configPaths.src + '**/*.test.js'
```

Comment:
Non-blocking: maybe we could have a comment here to say why the order of these is important (I saw that this was already discussed in the comments on the PR)

Reply:
Ah sorry @hannalaakso  it's been merged already.

If it helps, the order was already correct in these two:

```
./tasks/gulp/copy-to-destination.js
./tasks/gulp/sassdoc.js
```

For reference, latest docs are here:
https://github.com/gulpjs/glob-stream#globbing--negation

Seems to be performance related to now glob based on the specified order.